packages/cli-core/src/format/format.ts

```diff
@@ -16,8 +25,11 @@ export const format = {
   error: (message: string) => red(message),
   note: (message: string) => grey(message),
   command: (command: string) => cyan(command),
+  highlight: (command: string) => cyan(command),
   success: (message: string) => green(message),
   sectionHeader: (header: string) => bold(blue(header)),
+  bold: (message: string) => bold(message),
+  dim: (message: string) => dim(message),
```

Comment:
could we reuse `note` instead?

Reply:
are `grey` and `dim` the same?

Reply:
Just tried, dim is much more muted than grey. Keeping the same for now as requested by product.